fastfold/model/hub/alphafold.py

```diff
@@ -41,6 +41,7 @@
     tensor_tree_map,
 )
 
+from habana.hpuhelper import *
```

Comment:
This line may break on non-habana environment, maybe need `habana.is_habana()`

Reply:
will fix in updated patch